src/mixins/selection.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...selection({
+ *     'background': 'blue'
+ *   }, 'section')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${selection({'background': 'blue'}, 'section')}
```

Comment:
Uhh, I don't actually know. It might need to be `Object|Array<any>`? Have you tried that? /cc @relekang